packages/graphql/tests/schema/issues/3439.test.ts

```diff
@@ -21,7 +21,7 @@ import { printSchemaWithDirectives } from "@graphql-tools/utils";
 import { gql } from "graphql-tag";
 import { lexicographicSortSchema } from "graphql/utilities";
 import { Neo4jGraphQL } from "../../../src";
-import { TestSubscriptionsPlugin } from "../../utils/TestSubscriptionPlugin";
+import { TestSubscriptionsMechanism } from "../../utils/TestSubscriptionsMechanism";
```

Comment:
@darrellwarde, isn't that suppose to be in another PR? This PR is about the OGM `context` type

Reply:
Yes, it's just a broken test from a merge from `dev`, just taking the opportunity to fix it. 🙃

Reply:
fair enough 😄